### PR TITLE
Bugfix/categorical data fixes

### PR DESCRIPTION
**Pull request by sanketshahc**
Maximum 10 categories visible on map, For bar chart, just using stacked bar chart, with counties on x axis, # of tracts on y, and colored sections for the actual category. I felt this was the most intuitive / useful way to show it. 

Disabled graphing on the compare features chart for categorical data.

### CODE REVIEW

File: .streamlit/config.toml [hunk 1]
```diff
@@ -5,5 +5,5 @@ token='pk.eyJ1IjoiamFyZWRzdG9jayIsImEiOiJjam1jZnU5eTM1eG9iM3BwNDRmanhoNmh5In0.T4
 [logger]
 
 # Level of logging: 'error', 'warning', 'info', or 'debug'.
-# Default: 'info'
-level = "warning"
\ No newline at end of file
+# Default: 'debug'
+level = "debug"
```
Comment (jstock29) [inline .streamlit/config.toml:9]: Not a huge deal, but should set to warning or error when deploying to prod

File: visualization.py [hunk 2]
```diff
@@ -114,16 +129,64 @@ def make_correlation_plot(df: pd.DataFrame, default_cols=[]):
...
-    bar = alt.Chart(df).mark_bar() \
-        .encode(x='tract_id', y=feature + ':Q',
-                tooltip=['tract_id', feature])
+def make_census_chart(df: pd.DataFrame, feature: str):
+    feat_type = 'category' if df[feature].dtype == 'object' else 'numerical'
+    data_df = pd.DataFrame(df[[feature, 'tract_id', 'county_name']])
+    if feat_type == 'category':
+        data_df = pd.DataFrame(data_df.groupby(['county_name', feature]).size())
+        data_df = data_df.rename(columns={0: "tract count"})
+        print(data_df)
```
Comment (jstock29) [inline visualization.py:155]: remove prints for prod

**Comment by sanketshahc** [comment]
ready to pull again

**Review by jstock29 — COMMENTED** [review]
👍

**Review by jstock29 — APPROVED** [review]

**Pull request merged**